region/maridia/inner-pink/West Sand Pit.json

```diff
@@ -757,8 +771,15 @@
         "HiJump with a good jump from the sand can reach the Solid Rock Maze region.  Use the sandfall if Samus gets stuck in the sand.",
         "Perform several wall jumps to climb up to the water level, then precise wall jumps to break free."
       ]
+    },
+    {
+      "id": 3,
+      "name": "Right Item Collection",
+      "note": [
+        "Collecting the right side Item without Springball or explosives can be achieved with a turnaround aim cancel, stationary Morph turnaround or an X-Ray turnaround."
```

Comment:
```suggestion
        "Collecting the right side Item is possible by aligning Samus on the closest pixel to the item (while facing left) and then performing a turnaround towards the right.",
        "This can be achieved with a Morph turnaround, an X-Ray turnaround, or a small number of turnaround aim cancels."
```